### PR TITLE
Update outdated branching strategy

### PR DESCRIPTION
**Pull request by RCheesley**
We have changed the branching strategy but we did not update it here! 🤦‍♀️

### CODE REVIEW

File: pages/03.community-structure/01.Governance model/01.code-governance/docs.md [hunk 1]
```diff
@@ -278,9 +278,22 @@ If you are not familiar with [Semantic Versioning](https://semver.org/) terminol
...
 From Mautic 3.0, the following approach is taken:
 
-**master branch** is used for active development of Mautic (e.g. version 4). As soon as we “feature freeze” a new version (that is, all major new features are done and we’re ready to release an alpha or beta), we move development efforts to a release-x.x branch (e.g. release-3.0 for Mautic 3.0)
+Mautic follows [Semantic Versioning][semver], which is best illustrated by an example. Let's say we just released a 3.0.0 version of Mautic, the following would apply:
 
-**release-x.x branches** are used to maintain minor versions of Mautic after they have been released. For example, if we need to provide bug fixes for Mautic 2.16.0, we’d check out the release-2.16 branch on Git and start working from there. Also, if we’d decide to create a new minor version, like 2.17, and a new major version (like 3.0) has already been released, we’d use the release-2.16 branch as the starting point and create release-2.17 from there.
+|Mautic version|Breaking changes/features allowed?|New features/enhancements allowed?|Bug fixes allowed?|
+|---|---|---|---|
+|3.2.1|❌|❌|✅|
+|3.2.0|❌|✅|✅|
```
Comment (dennisameling) [inline pages/03.community-structure/01.Governance model/01.code-governance/docs.md:286]:
```suggestion
|3.3.0|❌|✅|✅|
```

File: pages/03.community-structure/01.Governance model/01.code-governance/docs.md [hunk 2]
```diff
@@ -278,9 +278,22 @@ If you are not familiar with [Semantic Versioning](https://semver.org/) terminol
 
 From Mautic 3.0, the following approach is taken:
 
-**master branch** is used for active development of Mautic (e.g. version 4). As soon as we “feature freeze” a new version (that is, all major new features are done and we’re ready to release an alpha or beta), we move development efforts to a release-x.x branch (e.g. release-3.0 for Mautic 3.0)
+Mautic follows [Semantic Versioning][semver], which is best illustrated by an example. Let's say we just released a 3.0.0 version of Mautic, the following would apply:
```
Comment (dennisameling) [inline pages/03.community-structure/01.Governance model/01.code-governance/docs.md:281]:
```suggestion
Mautic follows [Semantic Versioning][semver], which is best illustrated by an example. Let's say we just released a 3.2.0 version of Mautic, the following would apply:
```

**Review by dennisameling — CHANGES_REQUESTED** [review]
This looks great, just noticed some minor typos which I'll also fix in the original article at https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests#choose-the-right-branch!

**Comment by RCheesley** [comment]
Ignore my comment in the other PR - just read this one!

**Pull request merged**